src/pages/components/tile/style.mdx

```diff
@@ -28,99 +28,170 @@ and structure.
 
 ## Color
 
-<InlineNotification>
+### Base tile color
 
-The tile border applies to clickable, selectable and expandable tiles — not to
-base tiles.
+| Element   | Property         | Color token     |
+| --------- | ---------------- | --------------- |
+| Container | background-color | `$layer` \*     |
+| Text      | text color       | `$text-primary` |
 
-</InlineNotification>
+<Row>
+<Column colLg={8}>
+
+![Base tile color](images/tile-style-color-base.png)
+
+<Caption>Base tile color</Caption>
+
+</Column>
+</Row>
+
+### Clickable tile color
 
-| Element         | Property         | Color token         |
-| --------------- | ---------------- | ------------------- |
-| Tile:background | background-color | `$layer` \*         |
-| Tile border     | border           | `$border-tile` \*   |
-| Icon            | svg              | `$icon-interactive` |
-| Checkbox icon   | svg              | `$icon-primary`     |
-| Checkmark icon  | svg              | `$icon-primary`     |
-| Chevron icon    | svg              | `$icon-primary`     |
+Clickable tiles have an available
+[feature flag](https://carbondesignsystem.com/components/overview/feature-flag/).
+
+| Element   | Property         | Color token         |
+| --------- | ---------------- | ------------------- |
+| Container | background-color | `$layer` \*         |
+| Text      | text color       | `$text-primary`     |
+| Border    | border           | `$border-tile` \*   |
+| Icon      | svg              | `$icon-interactive` |
 
 <Caption>
-  * Denotes a contextual color token that will change values based on the layer
+  \* Denotes a contextual color token that will change values based on the layer
   it is placed on.
 </Caption>
 
 <Row>
 <Column colLg={8}>
+<Tabs>
+
+<Tab label="Flag enabled">
+
+![Clickable tile color with feature flag enabled](images/tile-style-color-clickable-flag-enabled.png)
+
+</Tab>
+
+<Tab label="Flag disabled">
 
-![State of the base tile](images/tile-style-3.png)
+![Clickable tile color with feature flag disabled](images/tile-style-color-clickable-flag-disabled.png)
+
+</Tab>
+
+</Tabs>
 
 </Column>
 </Row>
 
-<Caption>
-  The example shows layering tokens applied across layers in the Gray 100 theme.
-</Caption>
+#### Clickable tile interactive color
```

Comment:
Use this instead of the `####` and it will automatically do the spacing correctly with the table.
```suggestion
<Title>Clickable tile interactive color</Title> 
```

Reply:
Oh nice, thanks @aagonzales 
Fixed for other Selectable & Expandable tile interactive color aswell!